### PR TITLE
BG changes from red to blue

### PR DESCRIPTION
**Pull request by YmYuekun**
BG changes from red to blue

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
   return (
-      <div style={{position:"absolute",width:"100%",height:"100%",background:'red'}}>
+      <div style={{position:"absolute",width:"100%",height:"100%",background:'blue'}}>
           <input style={{marginBottom:"15px",display:"flex", alignItems:'center',margin:"auto" }}/>
           <p style={{width:'100%', textAlign:'center'}}>name</p>
       </div>
   )
-}
\ No newline at end of file
+}g
```
Comment (zxt999) [inline src/app/page.tsx]: 多余了·

Comment (zxt999) [reply src/app/page.tsx]: 0.0

Comment (zxt999) [reply src/app/page.tsx]: >.<

File: src/app/page.tsx
Comment (zxt999) [inline src/app/page.tsx:1]: xiba

**Comment by YmYuekun** [comment]
Hi

**Comment by zxt999** [comment]
> Hi

씨,씨발

File: src/app/page.tsx (hunk 1 above)
Comment (zxt999) [inline src/app/page.tsx]: 111 @YmYuekun 111

**Comment by zxt999** [comment]
> Hi
ur code like shit

**Comment by zxt999** [comment]
> Hi

shit

**Review by zxt999 — APPROVED** [review]

**Pull request merged**